src/utils/types.ts

```diff
@@ -66,8 +66,8 @@ export interface SubscriptionProcesses {
 
 export interface SubscriptionInstance {
     subscription_id: string;
-    parent_relations: SubscriptionInstanceParentRelation[];
-    children_relations: SubscriptionInstanceParentRelation[];
+    in_use_by_relations: SubscriptionInstanceParentRelation[];
+    depends_relations: SubscriptionInstanceParentRelation[];
```

Comment:
Is see why `dependant/dependent on` could also work (in this place it reads easier I think)

FWIW:
`depends_on`, `used_by` would probably increase readability in code constructs. So without _relations.
But I can see that's it's difficult to make it consistent. We already had several ways to describe: has_children, children relations/parent_subscriptions etc. There are a lot of different post -and suffixes

Reply:
yes, I do question if they are even used, am not finding it anywhere